src/Pages/SignUp/component/Phone.js

```diff
@@ -0,0 +1,166 @@
+import styled from 'styled-components';
+import Theme from '../../../Styles/Theme';
+import { API } from '../../../config';
+
+function Phone({ idata, form, setForm }) {
+  const history = useHistory();
+
+  const handleInput = num => {
+    const { name, value } = num.target;
+    setForm({ ...form, [name]: value });
+  };
+
+  const vaildator = (name, value) => {
+    if (name === 'phoneNum' && value.length === 11) {
+      return true;
+    }
+    if (name === 'identNum' && value.length === 4) {
+      return true;
+    }
+  };
+
+  const Succeed = name => {
+    if (name === 'phoneNum') {
+      fetch(`${API}/users/identify`, {
+        method: 'POST',
+        headers: {
+          'Content-type': 'application/json',
+        },
+        body: JSON.stringify({
+          // 승찬님
+          phoneNumber: form.phoneNum,
+        }),
+      })
+        .then(res => res.json())
+        .then(res => {
+          console.log(res);
+          alert('인증번호를 전송하였습니다.');
+        });
+    }
+    if (name === 'identNum') {
+      fetch(`${API}/users/identify`, {
+        method: 'POST',
+        headers: {
+          'Content-type': 'application/json',
+        },
+        body: JSON.stringify({
+          // 승찬님
+          identifyNumber: form.identNum,
+        }),
+      })
+        .then(res => res.json())
+        .then(res => {
+          console.log(res);
+          // 성공 (메세지 확인)
+          if (res['MESSAGE'] === 'SUCCESS') {
+            alert('본인인증 완료');
+            history.push({ pathname: '/join/signup' });
+          } else {
+            alert('인증번호를 확인해주세요.');
+          }
+        });
+    }
+  };
+
+  //  이런식으로 가능? 위에거 줄이고싶음
+  // const Succeed = name => {
+  //   fetch(`${API}/users/identify`, {
+  //     method: 'POST',
+  //     headers: {
+  //       'Content-type': 'application/json',
+  //     },
+  //     body: JSON.stringify({
+  //       // 승찬님
+  //       phoneNumber: form.phoneNum,
+  //       identifyNumber: form.identNum,
+  //     }),
+  //   })
+  //     .then(res => res.json())
+  //     .then(res => {
+  //       if (name === 'phoneNum') {
+  //         console.log(res);
+  //         alert('인증번호를 전송하였습니다.');
+  //       }
+  //       if (name === 'identNum') {
+  //         console.log(res);
+  //         // 성공
+  //         if (res['MESSAGE'] === 'SUCCESS') {
+  //           alert('본인인증 완료');
+  //           history.push({ pathname: '/join/signup' });
+  //         } else {
+  //           alert('인증번호를 확인해주세요.');
+  //         }
+  //       }
+  //     });
+  // };
+
+  return (
+    <Section>
+      <Title>{idata.title}</Title>
+      <InputDiv>
+        <Input
+          onChange={handleInput}
+          name={idata.name}
+          type="text"
+          maxLength={idata.maxLength}
+          id={idata.id}
+          // isCorrect={vaildator(idata.name, form[idata.name])}
+        />
+        <Btn
+          onClick={Succeed}
+          disabled={vaildator(idata.name, form[idata.name]) ? false : true}
```

Comment:
```suggestion
          disabled={!vaildator(idata.name, form[idata.name])}
```
validator가 리턴하는 값이 boolean이기 때문에 반대 케이스로 만들어주려면 `!`연산자로 충분해보입니다